components/PostChooser/editor-partials/modal/index.js

```diff
@@ -33,14 +31,17 @@ export const PostChooserModal = ( props ) => {
 		order: 'desc',
 	} );
 	const [ searchType, setSearchType ] = useState( 'recent' );
+	const [ selectedPostType, setSelectedPostType ] = useState(
+		postTypes && postTypes.length > 0 ? postTypes[ 0 ].value : 'post'
```

Comment:
This works, but it makes an assumption that the first post type in the array is the one to show first, right? 

What about setting either a new prop like `defaultPostType = post` so the developer implementing the PostChooser would pass in the list of postTypes and which is the default post type. Or we could add `isDefault: true` property on the `postTypes` array: 

```
postTypes = [
			{ label: __( 'Posts' ), value: 'post'  },
			{ label: __( 'Pages' ), value: 'page', isDefault: true },
		], // Default post types to search.
```